uniffi_bindgen/src/interface/mod.rs

```diff
@@ -141,25 +144,34 @@ impl<'ci> ComponentInterface {
     }
 
     /// List the definitions for every Enum type in the interface.
-    pub fn iter_enum_definitions(&self) -> Vec<Enum> {
-        self.enums.to_vec()
+    pub fn iter_enum_definitions(&self) -> Vec<Enum<'_>> {
+        self.enums
+            .iter()
+            .map(|e| Enum {
```

Comment:
Here again we see that we're synthesizing the aggregate pointer thingy on demand.